workshops/models.py

```diff
@@ -119,7 +119,9 @@ class Membership(models.Model):
                              on_delete=models.PROTECT)
 
     def __str__(self):
-        return "{} Membership of <{}>".format(self.variant, str(self.organization))
+        return "{} membership: {} <{}>".format(self.variant.title(),
+                                               self.organization.fullname,
+                                               self.organization.domain)
```

Comment:
You could have left `str(self.organization)` here because you changed `Organization.__str__`.